modules/ROOT/pages/acm-lightning-components.adoc

```diff
@@ -201,6 +215,16 @@ The application listing component shows applications that use an API.
 
 // check *New Application URL* description
 
+== Learn More Button
+
+The learn more button enables users to access a custom page with additional information about the API.
+
+* *API*: Choose the API to show.
+* *Button Label*: Set the button label. Default is `Learn more about this API`.
+* *Button Align*: Set button alignment to left, right, or center.
+* *API Name*: Choose "Append", "Prepend", or "None" to add the API name to the button label.
+* *API Version*: Choose "Append", "Prepend", or "None" to add the API version to the button label.
```

Comment:
A fewe suggestions:
The learn more button enables users to access a custom page with additional information about the API.

 * *API*: Choose the API to show. (same comment as above)
* *Button Label*: Specifies the button label. Default is `Learn more about this API`.
* *Button Align*: Sets button alignment to *left*, *right*, or *center*. (Of what? What will this be next to in the UI?)
API Name: Adds the API name to *Button Label*. Values include *Append*, *Prepend*, or *None*. What is the default here?
API Version: Adds the API version to the *Button Label*. Values include *Append*, *Prepend*, or *None*. What is the default here?

Reply:
Updated with these changes.